app/controllers/miq_ae_class_controller.rb

```diff
@@ -2456,6 +2462,12 @@ def ordered_domains_for_priority_edit_screen
     MiqAeDomain.order('priority DESC')
       .reject  { |d| d.priority == 0 }
       .collect { |d| d.editable? ? d.name : add_read_only_suffix(d, d.name) }
+    User.current_tenant.editable_domains
+      .collect { |d| d.editable? ? d.name : add_read_only_suffix(d.name,
+                                                                 User.current_tenant.editable_domains.include?(d),
```

Comment:
by definition, `d` is an `editable_domain`, so I don't think you need to then ask if d is in the `editable_domains`.